packages/graph/package.json

```diff
@@ -26,7 +26,6 @@
     "test": "graph test"
   },
   "dependencies": {
-    "@graphprotocol/graph-cli": "0.30.4",
```

Comment:
removing this cause it breaks on m1

More info here: https://discord.com/channels/438038660412342282/903628101819461632/1009174614192181342